### DISCUSSION TITLE
Suggestion: Remove `@semantic-release/npm` as direct dependency

### DISCUSSION THREAD

**Discussion by pataar** [semantic-release/semantic-release#2249] [Ideas] [answered]
## New feature motivation

As someone, and I won't be the only one, that doesn't use the `@semantic-release/npm` plugin, it adds a lot of extra dependencies. Especially because of the `npm` dependency of `@semantic-release/npm`. My request would be to make it optional to install it as plugin.

## New feature implementation

Remove `@semantic-release/npm` as direct dependency. Update the documentation to instruct to install it separately when needed.

**Comment by gr2m** [comment, marked as answer]
We plan to release `@semantic-release/core` which won't include any plugins, so people will be able to build their own semantic-release version for different use cases.

I cannot give you an ETA on it though, we don't have much time to work on semantic-release right now, and we want to finish the transition to ESM before taking on anything else

**Reply by pataar** [reply]
That sounds like a great solution
